[INFO   ] Logger: Record log in /storage/emulated/0/projects/StickMan_Clone(Dump)/parser/.kivy/logs/kivy_20-09-14_13.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.8.3 (default, Jun 15 2020, 02:19:10) 
[GCC 9.3.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/bin/python3"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 2.0 58bf738'>
[INFO   ] GL: OpenGL vendor <b'ARM'>
[INFO   ] GL: OpenGL renderer <b'Mali-400 MP'>
[INFO   ] GL: OpenGL parsed version: 2, 0
[INFO   ] GL: Texture max size <4096>
[INFO   ] GL: Texture max units <8>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[WARNING] Base: Unknown <android> provider
[INFO   ] Base: Start application main loop
[INFO   ] GL: NPOT texture support is available
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/projects/StickMan_Clone(Dump)/parser/specialanimation.py", line 98, in <module>
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/base.py", line 504, in runTouchApp
[WARNING] stderr:     EventLoop.window.mainloop()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/window/window_sdl2.py", line 747, in mainloop
[WARNING] stderr:     self._mainloop()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/window/window_sdl2.py", line 479, in _mainloop
[WARNING] stderr:     EventLoop.idle()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/base.py", line 342, in idle
[WARNING] stderr:     self.dispatch_input()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/base.py", line 327, in dispatch_input
[WARNING] stderr:     post_dispatch_input(*pop(0))
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/base.py", line 233, in post_dispatch_input
[WARNING] stderr:     listener.dispatch('on_motion', etype, me)
[WARNING] stderr:   File "kivy/_event.pyx", line 707, in kivy._event.EventDispatcher.dispatch
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/window/__init__.py", line 1402, in on_motion
[WARNING] stderr:     self.dispatch('on_touch_down', me)
[WARNING] stderr:   File "kivy/_event.pyx", line 707, in kivy._event.EventDispatcher.dispatch
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/core/window/__init__.py", line 1418, in on_touch_down
[WARNING] stderr:     if w.dispatch('on_touch_down', touch):
[WARNING] stderr:   File "kivy/_event.pyx", line 707, in kivy._event.EventDispatcher.dispatch
[WARNING] stderr:   File "widget.py", line 549, in kivy.uix.widget.Widget.on_touch_down
[WARNING] stderr:   File "kivy/_event.pyx", line 707, in kivy._event.EventDispatcher.dispatch
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/uix/behaviors/button.py", line 151, in on_touch_down
[WARNING] stderr:     self.dispatch('on_press')
[WARNING] stderr:   File "kivy/_event.pyx", line 707, in kivy._event.EventDispatcher.dispatch
[WARNING] stderr:   File "/storage/emulated/0/projects/StickMan_Clone(Dump)/parser/specialanimation.py", line 92, in <lambda>
[WARNING] stderr:     if btn1.y>=200:
[WARNING] stderr:   File "/storage/emulated/0/projects/StickMan_Clone(Dump)/parser/specialanimation.py", line 34, in resume
[WARNING] stderr:     self.is_paused = False
[WARNING] stderr: AttributeError: 'NoneType' object has no attribute 'me'
